x-pack/legacy/plugins/siem/public/components/load_more_table/translations.ts

```diff
@@ -6,18 +6,18 @@
 
 import { i18n } from '@kbn/i18n';
 
-export const LOADING = i18n.translate('xpack.siem.loadingMoreTable.loadingDescription', {
+export const LOADING = i18n.translate('xpack.siem.loadMoreTable.loading', {
```

Comment:
I am not an expert here but I thought we need to be more descriptive by adding if it was a `description` or `label` or `title` etc...

Reply:
I think generally speaking, you are correct. However, in this particular instance, appending the string `Description` doesn't provide any additional or beneficial context (since these are literally for the words `Loading` and `Load more`). Thoughts?

Reply:
I will refer to that https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#naming-convention

Reply:
👍 Thanks for the link. I'll be more verbose with the ID.